compiler/modules/nback.llvm/llvm.bal

```diff
@@ -179,7 +179,7 @@ public class FunctionDefn {
     final string functionName;
 
     private BasicBlock[] basicBlocks = [];
-    private int varCount = 0;
+    private string[] variableNames = [];
```

Comment:
Don't think this the right data structure.

Reply:
You should have a map from variable name to count.

Reply:
Fixed